.travis.yml

```diff
@@ -17,9 +17,9 @@ before_install: git fetch --unshallow
 sudo: required
 language: java
 jdk:
-  - openjdk7
+  - openjdk8
 cache:
   directories:
   - "$HOME/.m2"
 install: MAVEN_OPTS="-Xms1G -Xmx1G" mvn install --batch-mode -DskipTests=true -Dmaven.javadoc.skip=true -Dmaven.source.skip=true
-script: mvn install -DexcludedGroups="org.apache.drill.categories.SlowTest,org.apache.drill.categories.UnlikelyTest,org.apache.drill.categories.SecurityTest" -DforkCount=1 -DmemoryMb=2560 -DdirectMemoryMb=4608
+script: mvn install -DexcludedGroups="org.apache.drill.categories.SlowTest,org.apache.drill.categories.UnlikelyTest,org.apache.drill.categories.SecurityTest" -DforkCount=1 -DmemoryMb=2560 -DdirectMemoryMb=6008
```

Comment:
@vladimirtkach Why the change in direct memory for travis? Did you run into an issue with the previous setting?

Reply:
@ilooner yes , there was:
testFlattenUpperLimit(org.apache.drill.exec.physical.unit.TestOutputBatchSize)  Time elapsed: 260.527 sec  <<< ERROR!
java.lang.OutOfMemoryError: GC overhead limit exceeded

For now changed to previous value.